types/coin.go

```diff
@@ -31,5 +31,6 @@ const (
 	DefaultGasPrice = 20
 )
 
+// TODO: move to chain config struct
```

Comment:
address todo or create linear ticket?

Reply:
Do you want me to create an issue in the os repo? I added this comment here but didn't update the code because it is more relevant to the os repo instead of this one. Ideally, in the Evmos repo we should just ha e tests with 18 decimals